src/SFML/Window/Unix/InputImpl.cpp

```diff
@@ -181,8 +182,19 @@ void setMousePosition(Vector2i position, const WindowBase& relativeTo)
 
     if (const WindowHandle handle = relativeTo.getNativeHandle())
     {
+        // Determine if cursor image will need restored after warp
+        const auto* cursorImage = XFixesGetCursorImage(display.get());
+        const bool restoreCursor = std::getenv("WAYLAND_DISPLAY") && (cursorImage->width != 1 || cursorImage->height != 1 ||
```

Comment:
What is the significance of checking `!= 1` as well as the 24th bit of `pixels[0]`?